dev/scripts/src/alluxio.org/build-distribution/cmd/root.go

```diff
@@ -36,3 +37,9 @@ func checkRootFlags() error {
 	}
 	return nil
 }
+
+func additionalFlags(cmd *flag.FlagSet) {
```

Comment:
can you add documentation to this function as well?

Reply:
updated